#!/usr/bin/env python
PACKAGE = "tracker"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("position_error_p_xy", double_t, 0, "position p parameter for xy axis", 0.1, 0, 1)
gen.add("position_error_p_z", double_t, 0, "position p parameter for z axis", 0.1, 0, 1)

gen.add("velocity_error_p_xy", double_t, 0, "velocity p parameter for xy axis", 0.1, 0, 1)
gen.add("velocity_error_p_z", double_t, 0, "velocity p parameter for z axis", 0.1, 0, 1)

exit(gen.generate(PACKAGE, "pva_tracker", "PVA_Tracker"))